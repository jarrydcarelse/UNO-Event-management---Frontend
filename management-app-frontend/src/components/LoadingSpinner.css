.loading-container {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 200px;
  width: 100%;
}

.loading-spinner {
  width: 50px;
  height: 50px;
  border: 4px solid #f2f2f7;
  border-top: 4px solid #FF8FAB;
  border-radius: 50%;
  animation: spin 1s linear infinite;
}

@keyframes spin {
  0% { transform: rotate(0deg); }
  100% { transform: rotate(360deg); }
} 