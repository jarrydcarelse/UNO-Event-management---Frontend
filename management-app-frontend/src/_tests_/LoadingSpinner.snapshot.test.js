import React from 'react';
import { render } from '@testing-library/react';
import LoadingSpinner from '../components/LoadingSpinner';

describe('LoadingSpinner Snapshot', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<LoadingSpinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
