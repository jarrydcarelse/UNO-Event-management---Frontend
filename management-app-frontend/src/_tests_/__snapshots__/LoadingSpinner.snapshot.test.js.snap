// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner Snapshot matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="loading-container"
  >
    <div
      class="loading-spinner"
    />
  </div>
</DocumentFragment>
`;
